tutorials/editor/using_the_web_editor.rst

```diff
@@ -0,0 +1,120 @@
+
+- Chrome 68 or later
+- Firefox 79 or later
+- Edge 79 or later
+
+Opera and Safari are not supported yet. Safari may work in the future once
+proper threading support is added.
+
+**Mobile browsers are currently not supported.**
+
+The web editor supports both the GLES3 and GLES2 renderers, although GLES2 is
+recommended for better performance and compatibility with old/low-end hardware.
+
+.. note::
+
+    If you use Linux, due to
+    `poor Firefox WebGL performance <https://bugzilla.mozilla.org/show_bug.cgi?id=1010527>`__,
+    it's recommended to use a Chromium-based browser instead of Firefox.
+
+.. _doc_using_the_web_editor_limitations:
+
+Limitations
+-----------
+
+Due to limitations on the Godot or Web platform side, the following features
+are currently missing:
+
+- No C#/Mono support.
+- No GDNative support.
+- No way to download the project files as a ZIP archive.
+- No project exporting.
+- The editor won't warn you when closing the tab with unsaved changes.
+- No lightmap baking support. You can still use existing lightmaps if they were
+  baked with a native version of the Godot editor.
+
+The following features are unlikely to be supported due to inherent limitations
+of the Web platform:
+
+- No support for external script editors.
+- No support for Android one-click deploy.
+
+.. seealso::
+
+    See the
+    `list of open issues on GitHub related to the web editor <https://github.com/godotengine/godot/issues?q=is%3Aopen+is%3Aissue+label%3Aplatform%3Ahtml5+label%3Atopic%3Aeditor>`__ for a list of known bugs.
+
+Importing a project
+-------------------
+
+To import an existing project, the current process is as follows:
+
+- Specify a ZIP file to preload on the HTML5 filesystem using the
+  **Preload project ZIP** input.
+- Run the editor by clicking **Start Godot editor**.
+  The Godot project manager should appear after 10-20 seconds.
+  On slower machines or connections, loading may take up to a minute.
+- Click the **Import** button on the right, click **Browse** in the dialog that appears
+  and choose ``preload.zip``.
+- Click the **Browse** button next to the Project Installation Path. Double-click
+  the ``projects`` folder then click **Create Folder** in the top-right corner of
+  the file dialog.
+- Specify a name for the project folder (it doesn't have to match the ZIP archive's name).
+  Click **OK**.
+- Now that the folder has been created, click **Select Current Folder**.
+  If everything went right, the Project Installation Path should look like this:
+  ``/home/web_user/projects/<project folder>``.
+- Click **Import** and the project will open in the editor.
+
+.. attention::
+
+    It's important to place the project folder in ``/home/web_user/projects``.
+    If your project folder is placed outside ``/home/web_user/projects``, you will
+    lose your project when closing the editor!
+
+This import process will be streamlined in the future.
```

Comment:
note, this will change in RC2 already:
- The zip preload will start the project manager with the `Import Zip` popup.
- Persistence will be across the whole `/home/web_user/` folder.